gnome-base/gvfs/metadata.xml

```diff
@@ -16,6 +16,7 @@
   <use>
     <flag name="afp">Enables support for accessing AFP (Apple Filing Protocol) network shares</flag>
     <flag name="archive">Enables support for accessing files in archives transparently via <pkg>app-arch/libarchive</pkg></flag>
+    <flag name="burn">Enables support for CDs burner location</flag>
```

Comment:
I think global "cdr" USE flag is used in the tree for this 

Thanks

Reply:
Done in https://github.com/gentoo/gentoo/pull/35307/commits/61a06872d12c11d84a34bd2cf8fbbfa4c27a2c7a